upload-api/functions/ucan.js

```diff
@@ -27,25 +22,34 @@ const AWS_REGION = process.env.AWS_REGION || 'us-west-2'
 async function handlerFn(request) {
   const {
     INVOCATION_BUCKET_NAME: invocationBucketName = '',
-    TASK_BUCKET_NAME: taskBucketName = '',
```

Comment:
It was never used and me and @vasco-santos realized that it would introduce extra indirection which we could avoid by using new pathing in form of


```
task/invocation@message.in
task/revocation@message.out
```